packages/requisitions/src/CustomerRequisition/api/api.ts

```diff
@@ -1,96 +1,102 @@
 import {
-  UpdateCustomerRequisitionInput,
-  NameResponse,
-  RequisitionQuery,
-  RequisitionLineConnector,
-  OmSupplyApi,
-  ConnectorError,
+  SortBy,
+  FilterBy,
+  RequisitionSortFieldInput,
+  RequisitionNodeType,
+  UpdateResponseRequisitionInput,
+  UpdateResponseRequisitionStatusInput,
 } from '@openmsupply-client/common';
-import { Requisition } from '../../types';
 
-const otherPartyGuard = (otherParty: NameResponse) => {
-  if (otherParty.__typename === 'NameNode') {
-    return otherParty;
-  } else if (otherParty.__typename === 'NodeError') {
-    throw new Error(otherParty.error.description);
-  }
-
-  throw new Error('Unknown');
-};
-
-const requisitionGuard = (requisitionQuery: RequisitionQuery) => {
-  if (requisitionQuery.requisition.__typename === 'RequisitionNode') {
-    return requisitionQuery.requisition;
-  }
-
-  throw new Error('Could not find the requisition');
-};
-
-const linesGuard = (
-  requisitionLines: RequisitionLineConnector | ConnectorError
-) => {
-  if (requisitionLines.__typename === 'RequisitionLineConnector') {
-    return requisitionLines.nodes;
-  }
-
-  if (requisitionLines.__typename === 'ConnectorError') {
-    throw new Error('Error fetching lines for requisition');
-  }
+import {
+  getSdk,
+  ResponseRequisitionFragment,
+  ResponseRequisitionRowFragment,
+  ResponseRequisitionsQuery,
+} from './operations.generated';
 
-  throw new Error('Unknown');
-};
+export type ResponseRequisitionApi = ReturnType<typeof getSdk>;
 
 export const requisitionToInput = (
-  requisition: Partial<Requisition> & { id: string }
-): UpdateCustomerRequisitionInput => {
+  requisition: Partial<ResponseRequisitionFragment> & { id: string }
+): UpdateResponseRequisitionInput => {
   return {
     id: requisition.id,
-    orderDate: requisition.requisitionDate?.toISOString(),
-    otherPartyId: requisition.otherParty?.id,
+    // otherPartyId: requisition.otherParty?.id,
```

Comment:
the other party ID is not supported currently in the API.. though I'm not sure it should be.. maybe just for request requisitions